admin/translation-status/index.html

```diff
@@ -1,6 +1,3 @@
----
----
-
```

Comment:
Why do you remove [YAML front matter](https://jekyllrb.com/docs/front-matter/)?

If you show the file directly, I recommend to use rake commands described in [Preview Your Changes](https://github.com/ruby/www.ruby-lang.org#preview-your-changes).

Reply:
Fixed in https://github.com/ruby/www.ruby-lang.org/commit/d0cbdb32665864929c618ac1c80e223af31250c3 thanks